Gale Databases.js

```diff
@@ -1,190 +1,211 @@
 {
-	"translatorID": "e3748cf3-36dc-4816-bf86-95a0b63feb03",
-	"label": "Gale Databases",
-	"creator": "Sebastian Karcher",
-	"target": "^https?://go\\.galegroup\\.com/",
-	"minVersion": "3.0",
-	"maxVersion": "",
-	"priority": 100,
-	"inRepository": true,
-	"translatorType": 4,
-	"browserSupport": "gcsibv",
-	"lastUpdated": "2018-10-07 15:50:14"
+	'translatorID': 'e3748cf3-36dc-4816-bf86-95a0b63feb03',
+	'label': 'Gale Databases',
+	'creator': 'Jim Miazek',
+	'target': '^https?://[^?&]*(?:gale|galegroup|galetesting|ggtest)\.com(?:\:\d+)?/ps/',
+	'minVersion': '3.0',
+	'maxVersion': '',
+	'priority': 100,
+	'inRepository': true,
+	'translatorType': 4,
+	'browserSupport': 'gcsibv',
+	'lastUpdated': '2019-10-11 12:00:00'
```

Comment:
You need to revert these quote changes, which will break the translator. (For historical reasons, this is JSON, not JavaScript.)